api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -194,7 +194,7 @@ message Policy {
 }
 
 // Permission defines an action (or actions) that a principal can take.
-// [#next-free-field: 13]
```

Comment:
LGTM, can you add an entry to https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.30/v1.30.0? Thanks.

Reply:
done, also made some small changes to tests and unnecessary null check, since it works the same either way